### PR TITLE
Bare metal scale up/down doc cleanup

### PR DESCRIPTION
**Pull request by ptrivedi**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ spec:
   - count: 1     # increase this number to horizontally scale your data plane
 ```
 
-Next, you must ensure you have enough available hardware for the scale-up operation to function. Available hardware could have been fed to the cluster as extra hardware during a prior create command, or could be fed to the cluster during the scale-up process by providing the hardware CSV file to the upgrade cluster command (explained in detail below).
-For scale-down operation, you can skip directly to the upgrade cluster command.
+Next, you must ensure you have enough available hardware for the scale up operation to function. Available hardware could have been fed to the cluster as extra hardware during a prior create command, or could be fed to the cluster during the scale up process by providing the hardware CSV file to the upgrade cluster command (explained in detail below).
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md:32]:
```suggestion
Next, you must ensure you have enough available hardware for the scale-up operation to function. Available hardware could have been fed to the cluster as extra hardware during a prior create command, or could be fed to the cluster during the scale-up process by providing the hardware CSV file to the upgrade cluster command (explained in detail below).
```
I think those hyphens were correct. Do you disagree?

File: docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ spec:
...
+For scale down operation, you can skip directly to the [upgrade cluster command]({{< relref "baremetal-scale/#upgrade-cluster-command-for-scale-updown" >}}).
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md:33]:
```suggestion
For scale-down operation, you can skip directly to the [upgrade cluster command]({{< relref "baremetal-scale/#upgrade-cluster-command-for-scale-updown" >}}).
```

File: docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ description: >
 ---
 
 ### Scaling nodes on Bare Metal clusters
-When you are horizontally scaling your Bare Metal EKS Anywhere cluster, consider the number of nodes you need for your control plane and for your data plane.
+When you are horizontally scaling your bare metal EKS Anywhere cluster, consider the number of nodes you need for your control plane and for your data plane.
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md:11]:
```suggestion
When you are horizontally scaling your Bare Metal EKS Anywhere cluster, consider the number of nodes you need for your control plane and for your data plane.
```
I've been treating "Bare Metal" like a formal name.

**Review by chrisnegus — COMMENTED** [review]
Looks good. Just a couple of small things.

File: docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md [hunk 4]
```diff
@@ -50,24 +50,32 @@ eksa-worker1                    type=worker-group-1,v1alpha1.tinkerbell.org/owne
...
-eksctl anywhere generate hardware -z <hardware.csv> | kubectl apply -f -
+eksctl anywhere upgrade cluster -f cluster.yaml
 ```
 
 ### Autoscaling
 
 EKS Anywhere supports autoscaling of worker node groups using the [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/) and as a [curated package](../../../../reference/packagespec/cluster-autoscaler/).
 
-See [here](../../../../reference/clusterspec/optional/autoscaling/) for details on how to configure your cluster spec to autoscale worker node groups for autoscaling.
+See [here](../../../../reference/clusterspec/optional/autoscaling) for details on how to configure your cluster spec to autoscale worker node groups for autoscaling.
```
Comment (TerryHowe) [inline docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md:81]:
```suggestion
See [here](../../../../reference/clusterspec/optional/autoscaling/) for details on how to configure your cluster spec to autoscale worker node groups for autoscaling.
```

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**